pkg/sql/execinfrapb/processors_changefeeds.proto

```diff
@@ -81,6 +81,12 @@ message ChangeFrontierSpec {
   // changefeed at that timestamp.
   repeated roachpb.Span tracked_spans = 1 [(gogoproto.nullable) = false];
 
+  // Change frontier legacy checkpoint
+  optional cockroach.sql.jobs.jobspb.ChangefeedProgress.Checkpoint checkpoint = 6;
```

Comment:
We should remove this now that it's no longer used

Reminder: decrement the field number of the next field when you do